Net/src/Socket.cpp

```diff
@@ -79,9 +79,10 @@ int Socket::select(SocketList& readList, SocketList& writeList, SocketList& exce
 	if (epollSize == 0) return 0;
 
 	int epollfd = -1;
-	{
+	{		
 		struct epoll_event eventsIn[epollSize];
-		memset(eventsIn, 0, sizeof(eventsIn));
+		memset(eventsIn, 0, sizeof(epoll_event) * epollSize );
```

Comment:
gcc with -std=c++11 sizeof(eventsIn) gives bytes number
gcc with -std=c++14 sizeof(eventsIn) gives elements number
changed memset to keep it save in both condition